memcached.c

```diff
@@ -3379,8 +3396,13 @@ static inline void process_get_command(conn *c, token_t *tokens, size_t ntokens,
 
                 if (return_cas || !settings.inline_ascii_response)
                 {
-                  MEMCACHED_COMMAND_GET(c->sfd, ITEM_key(it), it->nkey,
+                  if (should_touch) {
```

Comment:
remove this branch and leave it at GET. should help with perf enough to get it over the line, and it's still more a get than a touch (which doesn't return the data). not sure I care enough for it to be semantically correct.